modules/raft/src/main/java/org/apache/ignite/raft/jraft/Node.java

```diff
@@ -178,18 +178,19 @@ public interface Node extends Lifecycle<NodeOptions>, Describer {
      * Change the configuration of the raft group to |newPeers| , done.run() would be invoked after this operation
      * finishes, describing the detailed result.
      *
-     * @param newPeers new peers to change
+     * @param newPeersAndLearners new peers and learners to change
+     * @param term term on which this method was called. If real raft group term will be different - changePeers will be skipped.
      * @param done callback
      */
-    void changePeers(final Configuration newPeers, final Closure done);
+    void changePeers(final Configuration newPeersAndLearners, long term, final Closure done);
```

Comment:
Params differ from `void changePeersAsync(final Configuration newConf,`. I would expect the same naming for both two methods.

Reply:
Fixed.